Main.qml

```diff
@@ -362,21 +342,66 @@ ApplicationWindow {
                         NumberAnimation { properties: "x,y"; easing.type: Easing.OutBounce; duration: animationDuration }
                     }
 
+                    function peekIssue(lineNumber, linePosition) {
+                        if (lineNumber === 0 && linePosition === 0) { return; }
+                        resultsPageEditor.state = "VISIBLE"
+                        resultsPageEditor.scrollToLine(lineNumber)
+                    }
+
+                    // this should be done better with findBlockByLineNumber from QTextDocument(), requires C#
+                    function lineNumberToY(lineNumber) {
+                        return resultsPageEditor.letterHeight * lineNumber;
+                    }
```

Comment:
This function isn't needed anymore, forgot to remove it

Reply:
Removed.